src/app/utils/field-validation/index.ts

```diff
@@ -41,6 +52,41 @@ const doFieldTypesMatch = (
     : right(undefined)
 }
 
+/**
+ * Returns true if response appears on a hidden field.
+ * This may happen if a submission is made programatically to try and bypass form logic.
+ * @param response The submitted response
+ */
+const isResponsePresentOnHiddenField = (
+  response:
+    | ISingleAnswerResponse
+    | ICheckboxResponse
+    | ITableResponse
+    | IAttachmentResponse,
```

Comment:
does a union of these types already exist?

Reply:
actually yes, `type FieldResponse`. will use that instead